src/utils/rfqm_health_check.ts

```diff
@@ -0,0 +1,50 @@
+export enum HealthCheckStatus {
+    Operational = 'operational',
+    Unknown = 'unknown',
+    Maintenance = 'maintenance',
+    Degraded = 'degraded',
+    Failed = 'failed',
+}
+
+interface HealthCheckIssue {
+    status: HealthCheckStatus;
+    description: string;
+}
+
+export interface HealthCheckResult {
+    status: HealthCheckStatus;
+    pairs: {
+        [pair: string]: HealthCheckStatus; // where the pair has the form `${contractA}-${contractB}`
+    };
+    http: {
+        status: HealthCheckStatus;
+        issues: HealthCheckIssue[];
+    };
+    workers: {
```

Comment:
would it make sense to make this an array or object so each worker is represented?

Reply:
Mmm...not a bad idea maybe. I'll keep it in mind when I start building that part out.